packages/console/src/pages/Applications/index.tsx

```diff
@@ -19,23 +28,35 @@ const Applications = () => {
       <table className={styles.table}>
         <thead>
           <tr>
-            <td className={styles.applicationName}>Application Name</td>
-            <td>Client ID</td>
+            <td className={styles.applicationName}>{t('applications.application_name')}</td>
+            <td>{t('applications.application_name')}</td>
           </tr>
         </thead>
         <tbody>
-          <tr>
-            <td>
-              <ItemPreview
-                title="Default App"
-                subtitle="Single Page Application"
-                icon={<ImagePlaceholder />}
-              />
-            </td>
-            <td>
-              <CopyToClipboard value="RUMatENw0rFWO5aGbMI8tY2Qol50eOg3" />
-            </td>
-          </tr>
+          {error && (
+            <tr>
+              <td colSpan={2}>error occurred: {error.metadata.code}</td>
+            </tr>
+          )}
+          {isLoading && (
+            <tr>
+              <td colSpan={2}>loading</td>
+            </tr>
+          )}
```

Comment:
Loading should block the entire table in order to avoid further interactions, isn't it?

Reply:
Discussed offline.